client/scss/components/_segmented_control.scss

```diff
@@ -0,0 +1,71 @@
+.segmented-control__drawer {
+  display: none;
+
+  .enhanced & {
+    display: block;
+
+    @include respond-to('medium') {
+      display: none;
+    }
+  }
+}
```

Comment:
Do we not have root `display` classes (JS and breakpoints?) <= if not we do this quite a lot so could astract it.

Reply:
We _do_ have `is-hidden-s/m/l/xl` classes, but they don't (and most likely shouldn't) account for whether or not the thing they're hiding is within an `enhanced` body, so they aren't equipped to deal with this progressively enhanced show/hiding.

Reply:
If I understand this correctly, the drawer is hidden - shouldn't we just create a class then called `.enhanced-visible` which is:

```SCSS
.enhenced-visible {
  display: none;
  .enhanced & { display: initial }
}
```

Then just apply the styles we already have as we'll only see them in enhanced mode...

Reply:
P.S. Bikeshedding.